app/src/main/res/layout/item_popup_row.xml

```diff
@@ -71,5 +71,19 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/popupField"
                 app:layout_constraintVertical_bias="0.975" />
+
+        <Spinner
```

Comment:
Add Spinner to the recycler view row layout(item_popup_row.xml) to display the all the possible values for the fields of type Coded Value Domains